exercises/ocr-numbers/ocr_numbers_test.rb

```diff
@@ -5,87 +5,87 @@
 
 # Common test data version: 80782b6
 class OcrNumbersTest < Minitest::Test
-  def test_recognizes_0
+  def test_Recognizes_0
     # skip
     assert_equal "0", OcrNumbers.convert(" _ \n| |\n|_|\n   ")
   end
 
-  def test_recognizes_1
+  def test_Recognizes_1
     skip
     assert_equal "1", OcrNumbers.convert("   \n  |\n  |\n   ")
   end
 
-  def test_unreadable_but_correctly_sized_inputs_return_?
+  def test_Unreadable_but_correctly_sized_inputs_return_?
```

Comment:
What to do about punctuation characters is another issue.

But this one is probably a job for a pre-processor, since it's problem dependent.
`s/"?"/question mark string/`